### PR TITLE
Add release of 1.6.0

### PR DESCRIPTION
**Pull request by lunny**

### CODE REVIEW

File: content/post/release-of-1.6.0.md [hunk 1]
```diff
@@ -0,0 +1,324 @@
...
+
+If you lack the time or knowledge to do any of the above, just using Gitea and sharing the word is enough to make us happy! One thing you can always do is to report any bugs you find on the [Gitea issue tracker](https://github.com/go-gitea/gitea/issues).
+
+Before opening an issue, read the [contribution guidelines about reporting bugs](https://github.com/go-gitea/gitea/blob/master/CONTRIBUTING.md#bug-reports). After opening an issue, try to stick around a while to answer any questions we might have. Replies greatly help us find the root cause of an issue.
+
+# Thanks
+
+This release would not have been possible without the pull requests from the following people:
+
+* [@0rzech](https://github.com/0rzech)
```
Comment (lafriks) [inline content/post/release-of-1.6.0.md]: I think there is missing more people... :)

**Comment by lafriks** [comment]
I hope you don't mind that I did some fixes

File: content/post/release-of-1.6.0.md (hunk 1 above)
Comment (lunny) [reply content/post/release-of-1.6.0.md]: Right, too sleepy so I add `WIP`

**Comment by lunny** [comment]
Could be reviewed and also I send a PR to go-gitea/changelog to add a subcommand to generate contributors list. https://github.com/go-gitea/changelog/pull/10

**Pull request merged**